### DISCUSSION TITLE
使用whisper语音转文本，出现报错 Model ggml-tiny.en.bin does not support fr-FR

### DISCUSSION THREAD

**Discussion by lishan256** [ZuodaoTech/everyone-can-use-english#732] [Q&A]
之前一直用法语练习，都没有问题。昨天还用过，都一切正常，今天用的时候，就有了这个报错。不知道什么情况……

![1719546878486](https://github.com/xiaolai/everyone-can-use-english/assets/89031156/b0cf0341-c8cd-42ee-8661-032b37d61497)

**Comment by an-lee** [comment]
我检查一下，升级版本之后出现的报错吗？

语音转文本服务 可以先选 AzureAI 来用。

**Reply by lishan256** [reply]
不是，今天和昨天没有升级，是前几天升级的，这几天版本没有变过。

刚试过Azure AI，等了5分钟，我一分半的音频转化成功了。

想请教一下，以后本地的whisper就是不能用了吗？

**Reply by an-lee** [reply]
你在设置里在 语音转文本 选本地，然后点一下 检查 按钮看看啥反应

**Reply by lishan256** [reply]
显示whisper正常工作。
![1719692831884](https://github.com/ZuodaoTech/everyone-can-use-english/assets/89031156/67b8a463-406f-408c-9004-5b7619c50539)

**Reply by an-lee** [reply]
那你再试试用 本地 转一下，应该能用。

**Reply by lishan256** [reply]
还是不行，出现同样地报错，说不支持法语。

**Reply by an-lee** [reply]
哦，我知道了。你的 whisper 模型要选一个不带 `.en` 的才行。带 `.en` 的模型只支持英文。

**Reply by lishan256** [reply]
那这个具体要怎么操作呀？我自己没有选whisper模型，都是默认设置。

**Reply by lishan256** [reply]
我找到了，选tiny，而不是tiny.en，然后选中，重启，之后就可以用了。

**Reply by lishan256** [reply]
非常感谢！
